validate.py

```diff
@@ -249,7 +249,7 @@ def validate_required(data, field, msg=''):
       errors.extend(validate_values(row, 'Closing Fiscal Year'))
       errors.extend(validate_values(row, 'Closing Quarter'))
 
-    if row.get('planned hours of facility availability') and int(row.get('planned hours of facility availability')) == 0:
+    if row.get('planned hours of facility availability') and int(float(row.get('planned hours of facility availability'))) == 0:
```

Comment:
`int('1.1')` throws a 'ValueError: invalid literal for int() with base 10', since these fields are being read in as strings and Python doesn't like it when you try to convert a string representation of a float into an int.  In order to handle that I double cast the values being read in.  You could also handle it with a try/except statement but I went this way as it seemed the less obtrusive route.

Reply:
We don't want to coerce types, though - if a float has been passed, that's an incorrect type and an error we need to add to the list. We need a separate test to make sure the values match the types we expect. E.g., you can't have `.1` of an hour or `.5` of a server rack for our purposes.

Reply:
I agree, this works since the units are in thousands of hours but we'd come into a problem if the field were a proportion etc.  Testing can also be done with the .isdigit() method since strings are being passed.  

I'm not sure how to proceed at this moment.  I implemented this fix after notifying the ME team to the error on May 6th and have applied it to every subsequent round of validation.  Adding an extra test is doable but may add extra warnings to agencies who already have a green light.

Reply:
We need to be continuously improving our validation process. I'm ok with our standards being increasingly tight with each passing quarter, and agencies need to be prepared to fix their data as we are able to increase our scrutiny. That's what we've been doing for the last two years, so this shouldn't come as a surprise.

Reply:
Also - this is way too late to fix and rollout this quarter, but we can prepare for Q3 2019.